meta-resin-common/recipes-connectivity/resin-ntp-config/resin-ntp-config.bb

```diff
@@ -0,0 +1,27 @@
+DESCRIPTION = "resin NTP configuration"
+SECTION = "console/utils"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+PR = "r0"
+
+SRC_URI = " \
+    file://resin-ntp-config \
+    file://resin-ntp-config.service \
+    "
+S = "${WORKDIR}"
+
+inherit allarch systemd
+
+SYSTEMD_SERVICE_${PN} = "resin-ntp-config.service"
+RDEPENDS_${PN} = "bash jq"
```

Comment:
I don't see any bash specific things in the script so I don't think we should depend on bash.